### PR TITLE
Data release 20240605

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/statvar/golden/search_statvar/count_person.json [hunk 1]
```diff
@@ -700,10 +700,18 @@
       "name": "Population: 15 Years or Less",
       "dcid": "Count_Person_15OrLessYears"
     },
+    {
```
Comment (shifucun) [inline internal/server/statvar/golden/search_statvar/count_person.json:703]: @chejennifer can we remove these stat var search tests?

File: internal/server/v1/observations/golden/bulk_series_linked/all_FRA_AA2_2016.json [hunk 2]
```diff
@@ -211,233 +211,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```
Comment (shifucun) [inline internal/server/v1/observations/golden/bulk_series_linked/all_FRA_AA2_2016.json:215]: why is this deletion/diff?

File: internal/server/v1/observations/golden/bulk_series_linked/preferred_FRA_AA2_2016.json [hunk 3]
```diff
@@ -93,136 +93,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```
Comment (shifucun) [inline internal/server/v1/observations/golden/bulk_series_linked/preferred_FRA_AA2_2016.json:97]: similar here, seems suspicious diff

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json [hunk 4]
```diff
@@ -8442,7 +8442,6 @@
         {
           "name": "Oslo",
           "types": [
-            "AdministrativeArea3",
```
Comment (shifucun) [inline internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json:8445]: @n-h-diaz is there some place import group change?

File: internal/server/v1/triples/golden/bulk_triples_in/result2.json [hunk 5]
```diff
@@ -4017,6 +4017,54 @@
       "triples": {
         "measurementDenominator": {
           "nodes": [
+            {
```
Comment (shifucun) [inline internal/server/v1/triples/golden/bulk_triples_in/result2.json:4020]: I think the token here https://github.com/datacommonsorg/mixer/blob/a91ea1008f050776dfec7fabe9d502df1c59e233/internal/server/v1/triples/golden/bulk_triples_in_test.go#L51 does not match the one in https://github.com/datacommonsorg/mixer/blob/a91ea1008f050776dfec7fabe9d502df1c59e233/internal/server/v1/triples/golden/bulk_triples_in/result1.json.

can you update the token in test.go to match the actual token? We have a few similar places, where you can search for "H4sIAAAAAAAA" in`*test.go` and make sure they all match the actual one from `*1.json` file

File: internal/server/v1/observations/golden/bulk_series_linked/all_FRA_AA2_2016.json (hunk 2 above)
Comment (hareesh-ms) [reply internal/server/v1/observations/golden/bulk_series_linked/all_FRA_AA2_2016.json:215]: This is due to CL/638945277 in which the NUTS<N> to AA<N> bulk change was reverted.
Team is mapping the AdministrativeArea as per the AA level specified in Wikipedia for individual countries as part of boundary addition now.

File: internal/server/v1/observations/golden/bulk_series_linked/preferred_FRA_AA2_2016.json (hunk 3 above)
Comment (hareesh-ms) [reply internal/server/v1/observations/golden/bulk_series_linked/preferred_FRA_AA2_2016.json:97]: cl/638945277 - Revert NUTS<N> to AA<N> mapping.

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json (hunk 4 above)
Comment (hareesh-ms) [reply internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json:8445]: Related to cl/638945277 : Revert NUTS to AA mapping.

File: internal/server/v1/triples/golden/bulk_triples_in/result2.json (hunk 5 above)
Comment (hareesh-ms) [reply internal/server/v1/triples/golden/bulk_triples_in/result2.json:4020]: Updated the token in script and regenerated the goldens.

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Review by pradh — APPROVED** [review]

**Pull request merged**